classes/plugins/ImportExportPlugin.php

```diff
@@ -412,26 +425,35 @@ public function getBounceTab($request, $title, $bounceUrl, $bounceParameterArray
      *
      * @param string $exportFileName
      */
-    public function downloadExportedFile($exportFileName)
+    public function downloadExportedFile(string $exportFileName, string $exportFilePath, PKPImportExportDeployment $deployment)
     {
+        $date = DateTime::createFromFormat(self::EXPORT_FILE_DATE_PART_FORMAT,  $exportFilePath);
```

Comment:
I'm finding `$exportFilePath` to be confusing; it's actually supposed to be a string-formatted date, not a path. Elsewhere it's called `$filename`, elsewhere `$downloadPath`. I'd recommend standardizing on something sensible, proabably relating to the date.

Reply:
Yes, completely agreed. Changed.